### PR TITLE
Added image dic for root menu items #7815

### PR DESCRIPTION
**Pull request by gautampanday**

### CODE REVIEW

File: cms.py [hunk 1]
```diff
@@ -227,6 +227,7 @@ def get_menu_item(self, max_depth):
         if self.type_ == 'record':
             res['record'] = self.record
             res['link'] = self.record.get_absolute_url()
+            res['image'] = self.record.image
```
Comment (aroraumang) [inline cms.py]: There are no images associated to menuitems

**Comment by aroraumang** [comment]
LGTM

File: cms.py [hunk 2]
```diff
@@ -90,6 +90,7 @@ class MenuItem(ModelSQL, ModelView, CMSMenuItemMixin):
     __name__ = 'nereid.cms.menuitem'
     _rec_name = 'title'
 
+    image = fields.Many2One('nereid.static.file', 'Image')
```
Comment (tarunbhardwaj) [inline cms.py]: Not needed

File: cms.py [hunk 3]
```diff
@@ -220,6 +221,7 @@ def get_menu_item(self, max_depth):
             'title': self.title,
             'target': self.target,
             'type_': self.type_,
+            'image': self.image,
```
Comment (tarunbhardwaj) [inline cms.py]: image: None

File: cms.py [hunk 4]
```diff
@@ -228,6 +230,9 @@ def get_menu_item(self, max_depth):
             res['record'] = self.record
             res['link'] = self.record.get_absolute_url()
 
+            if self.record.image:
+                res['image'] = self.record.image
```
Comment (tarunbhardwaj) [inline cms.py]: better call self.record.get_menu_item(max_depth=0) and update res

**Pull request closed without merging**